lib/straight-server/order.rb

```diff
@@ -99,8 +99,9 @@ def amount_paid_in_btc
       amount_in_btc(field: amount_paid, as: :string)
     end
 
+    TRANSACTION_AMOUNT_MINIMUM = 547
     def amount_to_pay_in_btc
-      amount_in_btc(field: amount.to_i - amount_paid.to_i, as: :string)
+      amount_in_btc(field: [amount.to_i - amount_paid.to_i, TRANSACTION_AMOUNT_MINIMUM].max, as: :string)
```

Comment:
You should use variable for this code:
`[amount.to_i - amount_paid.to_i, TRANSACTION_AMOUNT_MINIMUM].max`